docs/conf.py

```diff
@@ -0,0 +1,39 @@
+import os
+import sys
+
+sys.path.append(os.path.abspath(".."))
+
+extensions = [
+    "myst_parser",
+    "sphinxawesome_theme.highlighting",
+
+    "sphinx.ext.autodoc",
+    "sphinx.ext.intersphinx",
+    "sphinx.ext.napoleon",
+    "sphinxcontrib_trio",
+]
+
+myst_enable_extensions = [
+    "attrs_inline",
+    "colon_fence",
+    "linkify",
+    "tasklist",
+]
+
+autodoc_typehints = "none"
+autodoc_member_order = "alphabetical"
+autodoc_default_options = {
+    "members": True,
+    "show-inheritance": True
+}
+
+html_theme = "sphinxawesome_theme"
+
+set_type_checking_flag = True
+source_suffix = ".md"
+
+md_prolog = """
+```{eval-rst}
+.. :rocker: replace:: 🚀
```

Comment:
Why do you replace only the rocket ?

Reply:
oh that was a test lol, didnt work, i replaced all emojis directly in readme cause sphinxemoji doesnt support the myst parser